src/util/psbt/map/input.rs

```diff
@@ -310,11 +310,11 @@ impl Map for Input {
         }
 
         impl_psbt_get_pair! {
-            rv.push(self.tap_key_sig as <PSBT_IN_TAP_KEY_SIG, _>|<Vec<u8>>)
+            rv.push(self.tap_key_sig as <PSBT_IN_TAP_KEY_SIG, _>|<ShnorrSig>)
```

Comment:
In 550948e1a632105a9d91a79f4fe415fb2b3c99ee:

`Schnorr` is spelled wrong. Lol, we really to clean up these macros.